avtozip/store/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from .models import Product
 
 # Register your models here.
```

Comment:
I would suggest to remove this comment from here and add additional lines between imports and actual code.